WcaOnRails/db/structure.sql

```diff
@@ -49,6 +49,7 @@ CREATE TABLE `Competitions` (
   `registration_close` datetime DEFAULT NULL,
   `use_wca_registration` tinyint(1) NOT NULL DEFAULT '0',
   `guests_enabled` tinyint(1) NOT NULL DEFAULT '1',
+  `results_posted` datetime DEFAULT NULL,
```

Comment:
Can we call this `results_posted_at` as per the `updated_at` and `created_at` fields?